### PR TITLE
Fix Splash screen

### PR DESCRIPTION
**Pull request by BurningAXE**
Updated some constraints and renamed the screen to reflect the design

### CODE REVIEW

**Review by nkumarcc — APPROVED** [review]
I'm a fan. How does it look with the action bar now? Just watch the merge conflicts and I think this can go in

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -4,11 +4,13 @@
     <color name="colorIsCurrentUserNotSick">#4CAF50</color>
 
     <!-- Imported from UX Repo -->
-    <color name="colorPrimary">#BE3E49</color>
-    <color name="colorSecondary">#FFFFFF</color>
+    <color name="colorPrimary">#2c58b1</color>
```
Comment (nkumarcc) [inline app/src/main/res/values/colors.xml:7]: James added named color specs, I think we should source `colorPrimary` from those (I think you'll see it when merging with `develop`)

**Comment by BurningAXE** [comment]
> How does it look with the action bar now?

Looks like a real app :D

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (BurningAXE) [reply app/src/main/res/values/colors.xml:7]: Ok, just overwrite my change when merging.

File: app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt [hunk 2]
```diff
@@ -5,11 +5,9 @@ import androidx.fragment.app.Fragment
...
 import android.view.View
 import android.view.ViewGroup
-import android.widget.TextView
 import androidx.databinding.DataBindingUtil
 import androidx.navigation.findNavController
-import org.covidwatch.android.databinding.FragmentTitleBinding
-import org.covidwatch.android.R
+import org.covidwatch.android.databinding.FragmentSplashBinding
 
 // TODO: Rename parameter arguments, choose names that match
```
Comment (Apisov) [inline app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt:12]: Please, remove template code from fragments.

**Review by Apisov — CHANGES_REQUESTED** [review]
I still need to finish the review but the whole implementation of the onboarding could have use some changes. I'll finish review tomorrow.

**Comment by BurningAXE** [comment]
Dropping this one favor of another

**Pull request closed without merging**